src/app/app-layout/app-header.component.ts

```diff
@@ -89,9 +95,8 @@ export class AppHeaderComponent implements OnInit {
       this.router.events
       .pipe(
         filter(event => event instanceof NavigationEnd),
-        map(() => {
-
-          // this.isHomePage = (event.url == '/');
+        map((event: NavigationEnd) => {
```

Comment:
Here, event would be null within the context of the map function - so just needed to pass it in